_data/config.yml

```diff
@@ -1,2 +1,3 @@
 grpc_release_branch: release-0_14
+grpc_java_release_tag: v1.0.0-pre1
```

Comment:
I considered it. You can see I used to have v1.0.0 hard-coded. I figured making it point to pre1 since now it is easy to bump the version, and it would help to have something copy/pasteable, and grpc_release_branch will also need to be bumped.